File: armstrong.cpp
#include<stdio.h>
int main()
{
	int k,a,b,c=0,d,e,f,g,h=0;
	printf("Enter a number:");
	scanf("%d",&a);
	b=a;
	while(b!=0)
	{
		b=b/10;
		c++;
	}
	
	for(d=1;d<=c;d++)
	{
	e=a%10;
	g=e;
		for(f=1;f<c;f++)
		{
			e=g*e;	
		}
	h=h+e;
	a=a/10;
	}

	if(h==k)
	{printf("%d is armstrong number!",h);
	}
	
	else
	{
		printf("%d is not an armstrong number.",h);
	}
}

File: armstrong.py
def is_armstrong(num):
    num_str= str(num)
    num_digits= len(num_str)
    print(num_digits)
    sum_power=0
    for digits in num_str:
        sum_power= sum_power + int(digits)**num_digits
    return sum_power  
    
    
  
n= int(input("Enter a number: "))
p=is_armstrong(n) 
if n==p:
    print("Armstrong")
    
else:
    print("Not Armstromg")    
    